# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/parent/FileSystemManagerParentFactory.cpp
# Commit: e4b3f3d8ba5f
# Full Hash: e4b3f3d8ba5fd951bf79a2014dcee109255c865f
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 1
# Description:
#   Bug 1786465 - Rename POriginPrivateFileSystem to PFileSystemManager; r=dom-storage-reviewers,jesup
#   
#   The child actor will be held by FileSystemManager on the child side and the
#   parent actor already holds FileSystemDataManager, so it makes sense to rename
#   the protocol to PFileSystemManager.
# ==============================================================================

diff -r 3b0da4b4bb83 -r e4b3f3d8ba5f dom/fs/parent/FileSystemManagerParentFactory.cpp
--- a/dom/fs/parent/FileSystemManagerParentFactory.cpp	Wed Aug 31 09:34:48 2022 +0000
+++ b/dom/fs/parent/FileSystemManagerParentFactory.cpp	Wed Aug 31 09:34:48 2022 +0000
@@ -7,8 +7,8 @@
 #include "FileSystemManagerParentFactory.h"
 
 #include "mozilla/StaticPrefs_dom.h"
+#include "mozilla/dom/FileSystemManagerParent.h"
 #include "mozilla/dom/FileSystemTypes.h"
-#include "mozilla/dom/OriginPrivateFileSystemParent.h"
 #include "mozilla/dom/quota/QuotaCommon.h"
 #include "mozilla/dom/quota/QuotaManager.h"
 #include "mozilla/ipc/Endpoint.h"
@@ -50,7 +50,7 @@
 
 mozilla::ipc::IPCResult CreateFileSystemManagerParent(
     const mozilla::ipc::PrincipalInfo& aPrincipalInfo,
-    mozilla::ipc::Endpoint<POriginPrivateFileSystemParent>&& aParentEndpoint,
+    mozilla::ipc::Endpoint<PFileSystemManagerParent>&& aParentEndpoint,
     std::function<void(const nsresult&)>&& aResolver) {
   QM_TRY(OkIf(StaticPrefs::dom_fs_enabled()), IPC_OK(),
          [aResolver](const auto&) { aResolver(NS_ERROR_DOM_NOT_ALLOWED_ERR); });
@@ -88,8 +88,8 @@
   InvokeAsync(taskqueue, __func__,
               [origin, parentEp = std::move(aParentEndpoint), aResolver, rootId,
                data = std::move(data), taskqueue, pbackground]() mutable {
-                RefPtr<OriginPrivateFileSystemParent> parent =
-                    new OriginPrivateFileSystemParent(taskqueue, rootId);
+                RefPtr<FileSystemManagerParent> parent =
+                    new FileSystemManagerParent(taskqueue, rootId);
                 if (!parentEp.Bind(parent)) {
                   return BoolPromise::CreateAndReject(NS_ERROR_FAILURE,
                                                       __func__);